src/pybind/mgr/cephadm/tlsobject_store.py

```diff
@@ -205,39 +211,72 @@ def list_tlsobjects(self) -> List[Tuple[str, TLSObjectProtocol, Optional[str]]]:
         return tlsobjects
 
     def load(self) -> None:
+
+        def _kv_preview(key: str, raw: str, n: int = 20) -> Tuple[str, int, str]:
+            # Safe, short, escaped preview for logs
+            return key, (len(raw) if raw else 0), (raw[:n] if raw else "")
+
         for k, v in self.mgr.get_store_prefix(self.store_prefix).items():
```

Comment:
Is the idea with the tombstones just that they'll show up as names in this loop when it does the `get_store_prefix` call?